### PR TITLE
Honk your clown nose

### PR DESCRIPTION
**Pull request by VictoriaActual**
#### Summary
Content "Honk your clown nose!"

#### Purpose of change

Clown noses couldn't be honked, which devastated me. ~~In addition compressed air horns, whistles, and a few other items don't use EOC so I figured I'd swap those over as well.~~ Nope, just updating the clown nose now

#### Describe the solution

~~Adds a new EOC that will replace all uses of 'manualnoise' that exist, and remove it once all the cleanup is done.~~
Clown nose now uses 'manualnoise' much like the whistle and compressed air horn.

#### Describe alternatives you've considered

The clown nose must honk, there is no alternative.

#### Testing

The current two commits have been tested by spawning in a whistle and using the item, which worked. Same thing with spawning in the clown nose and giving it a honk. :D

I reverted both of those and was able to test the clown nose in the evac shelter. It honks and emits noise 10 (slightly less than the bicycle horn)

#### Additional context

Thanks to @GuardianDll for suggesting using an EOC to implement the clown nose honking. Please let me know if I should instead split this PR into two parts: one handling removing the 'manualnoise' occurrences with the new EOC and a second updating the clown nose, or if this is fine as is.
I'm submitting this as a draft so I can finish it tomorrow after I get some sleep.

### CODE REVIEW

**Comment by GuardianDll** [comment]
oh, so we have json function for this 
didn't know that, but it doesn't matter now, since EoC is made already
good job

**Review by anothersimulacrum — CHANGES_REQUESTED** [review]
> Adds a new EOC that will replace all uses of 'manualnoise' that exist, and remove it once all the cleanup is done.

Don't do this, just use `manualnoise`.

File: data/json/items/armor/misc.json [hunk 1]
```diff
@@ -22,13 +22,22 @@
     "id": "clown_nose",
     "type": "ARMOR",
     "name": { "str": "clown nose" },
-    "description": "A red latex nose often worn by clowns.",
+    "description": "A red latex nose often worn by clowns. Squeezing the nose makes it honk.",
```
Comment (anothersimulacrum) [inline data/json/items/armor/misc.json:25]:
```suggestion
    "description": "A red latex nose often worn by clowns.  Squeezing the nose makes it honk.",
```

**Review by anothersimulacrum — APPROVED** [review]

**Comment by Maleclypse** [comment]
I was explaining the development process of DDA to a work friend today and honestly this PR is one of the most accurate embodiments of that process.  "I found a weird thing that you can do in the real world, and it could not be done in DDA.  This will not stand."  Thank you!

**Pull request merged**